types/wx-js-sdk-dt/index.d.ts

```diff
@@ -0,0 +1,468 @@
+// Type definitions for wx-js-sdk 1.2
+// Project: https://res.wx.qq.com/open/js/jweixin-1.2.0.js
+// Definitions by: Bian Zhongjie <https://github.com/agasbzj>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export namespace wx {
```

Comment:
/// <reference types="wx-js-sdk-dt" />

types name should be same as the folder name.

---

The tests here are fake, only check types and no runtime test, just write your use case for testing, they won't actually be executed.